Source/Scene/ModelExperimental/ModelColorPipelineStage.js

```diff
@@ -52,21 +51,9 @@ ModelColorPipelineStage.process = function (
 
   // Pass the model's color as a uniform. Set the pass type to translucent, if needed.
   const color = model.color;
-  const renderStateOptions = renderResources.renderStateOptions;
-  if (color.alpha === 0.0 && !model.hasSilhouette(frameState)) {
-    // When the model is invisible, only disable the color and depth writes
-    // if the model does not have a silhouette. Otherwise, the silhouette
-    // will not render.
-    renderStateOptions.colorMask = {
-      red: false,
-      green: false,
-      blue: false,
-      alpha: false,
-    };
-    renderStateOptions.depthMask = false;
```

Comment:
I removed this code because setting `depthMask` to false already happens in the `AlphaPipelineStage`, and setting `colorMask` to false is more like an optimization that's not strictly necessary.